content/webapp/components/Tabs/Tabs.styles.tsx

```diff
@@ -5,19 +5,19 @@ import { classNames, font } from '@weco/common/utils/classnames';
 export const Wrapper = styled.div`
   ${props =>
     `
-      margin: 0 -${props.theme.containerPadding.small}px; 
+      margin: 0 -${props.theme.containerPadding.small}px;
```

Comment:
It wasn't clear to me why these had diverged, but I don't think it was important enough to keep, so I just picked one